python/lsst/ap/pipe/make_apdb.py

```diff
@@ -116,8 +116,27 @@ def makeApdb(args=None):
     parser = ConfigOnlyParser()
     parsedCmd = parser.parse_args(args=args)
 
-    daxApdb.Apdb.makeSchema(parsedCmd.config)
-    apdb = daxApdb.make_apdb(config=parsedCmd.config)
+    # `make_apdb` is be replaced by `apdb-cli` commands, for now we keep it for
+    # backward compatibility, but only support SQL implementation here.
```

Comment:
If you want to remove `make_apdb.py`, I suggest emitting a deprecation warning when it's run.